package.json

```diff
@@ -211,6 +211,7 @@
     "modernizr": "^3.6.0",
     "modernizr-loader": "^1.0.1",
     "nyc": "^11.1.0",
+    "prop-types": "^15.6.1",
```

Comment:
No more deprecation warning 👍

Reply:
Eh, I think that's still there until we update some other dependencies.